packages/components/Callout/src/Callout.types.ts

```diff
@@ -212,6 +228,13 @@ export interface ICalloutProps extends IViewProps, ICalloutTokens {
    * anchoring to a point inside the component.
    */
   target?: React.RefObject<React.Component> | string;
+
+  /**
+   * Determines the material of the NSVisualEffectView that is the contentView of the Callout.
```

Comment:
After our discussion yesterday, did you find that we couldn't rely on the content to set the NSVisualEffectView because having a default at one material value e.g. `menu` was [multiplicative or otherwise not compatible] with the content being able to achieve the desired non-default material effect e.g. `popover`?

Reply:
I haven't tested that yet, I'll spin up a test app and see. It will depend on the [blending mode](https://developer.apple.com/documentation/appkit/nsvisualeffectview#1674173) of the two VisualEffectViews in question. In general, that feels a bit like undefined-behavior-land and once that could change between OS releases (It wouldn't be the first time), too I'm not a fan of a solution that involves placing two NSVisualEffectViews on top of each other.

Reply:
OK, I did some testing. There are two blending modes: `.behindWindow` and `.withinWindow`. I have a long vertical red column, with two VisualEffectViews on top to test what happens:

## 1) behindWindow

<img width="684" alt="Screenshot 2023-08-08 at 4 31 16 PM" src="https://github.com/microsoft/fluentui-react-native/assets/6722175/f90d365b-403d-4f68-9690-d803a8749120">

VisualEffectView ignores the view heirarchy and blends content directly behind the window. This means we can layer as many VisualEffectViews as we want, and is the default that Callout uses currently.

## 2) withinWindow 

<img width="539" alt="Screenshot 2023-08-08 at 4 31 57 PM" src="https://github.com/microsoft/fluentui-react-native/assets/6722175/7b865a65-426a-4012-b520-4077fc4e5864">

There is some multiplicativeness happening, which is undesirable.

Reply:
Conclusion: It seems that for VisualEffectViews with `.behindWindow` blending (the Callout default), you CAN stack them and the top-most one wins. That being said, I would still like to add this prop to control the Callout's root VisualEffectView.. I don't like relying on the stacking behavior + feels weird to need to add a new view when I could just modify the existing one.

Reply:
Gonna go with closing this PR and making https://github.com/microsoft/fluentui-react-native/pull/3026 instead